package.json

```diff
@@ -153,47 +153,47 @@
     },
     "configurations": {
       "iphone11.sim": {
-        "binaryPath": "ios/build/Build/Products/Release-iphonesimulator/COVIDSafePaths.app",
+        "binaryPath": "ios/build/Build/Products/Release-iphonesimulator/GPS.app",
```

Comment:
`test.yml` needs to be updated too:

https://github.com/Path-Check/covid-safe-paths/blob/develop/.github/workflows/test.yml#L217-L220